internal/api/graphql/graph/model/models.go

```diff
@@ -224,6 +224,33 @@ func NewIssueEntity(issue *IssueInput) entity.Issue {
 	}
 }
 
+func NewScannerRunEntity(sr *ScannerRunInput) entity.ScannerRun {
+
+	return entity.ScannerRun{
+		RunID:     -1,
+		UUID:      lo.FromPtr(sr.UUID),
+		Tag:       lo.FromPtr(sr.Tag),
+		Completed: false,
+		StartRun:  time.Now(),
+		EndRun:    time.Now(),
+	}
+}
+
+func NewScannerRun(sr *entity.ScannerRun) ScannerRun {
+	startRun := sr.StartRun.Format(time.RFC3339)
+	endRun := sr.EndRun.Format(time.RFC3339)
+
+	return ScannerRun{
+		ID:        fmt.Sprintf("%d", sr.RunID),
+		UUID:      sr.UUID,
```

Comment:
What distinguishes an ID from an UUID? What should each field track?

Reply:
The ID is just a normal ID used to identify the record.
The UUID is a random UUID selected by a Scanner to uniquely identify a ScannerRun w/o the need of a roundtrip to the database. This will enable the parallel execution of Scanners and make the whole protocol more fault tolerant for distributed systems.

Reply:
Ok, got it. Thx for the explanations.

Reply:
Ok, got it. Thx for the explanations.